Exdi/exdigdbsrvsample/GdbSrvControllerLib/exdiConfigData.xml

```diff
@@ -1,7 +1,315 @@
+          <Entry Name ="cpsr" Order = "21" Size = "8" />
+          <Entry Name ="V0" Order = "22" Size = "16" />
+          <Entry Name ="V1" Order = "23" Size = "16" />
+          <Entry Name ="V2" Order = "24" Size = "16" />
+          <Entry Name ="V3" Order = "25" Size = "16" />
+          <Entry Name ="V4" Order = "26" Size = "16" />
+          <Entry Name ="V5" Order = "27" Size = "16" />
+          <Entry Name ="V6" Order = "28" Size = "16" />
+          <Entry Name ="V7" Order = "29" Size = "16" />
+          <Entry Name ="V8" Order = "2a" Size = "16" />
+          <Entry Name ="V9" Order = "2b" Size = "16" />
+          <Entry Name ="V10" Order = "2c" Size = "16" />
+          <Entry Name ="V11" Order = "2d" Size = "16" />
+          <Entry Name ="V12" Order = "2e" Size = "16" />
+          <Entry Name ="V13" Order = "2f" Size = "16" />
+          <Entry Name ="V14" Order = "30" Size = "16" />
+          <Entry Name ="V15" Order = "31" Size = "16" />
+          <Entry Name ="V16" Order = "32" Size = "16" />
+          <Entry Name ="V17" Order = "33" Size = "16" />
+          <Entry Name ="V18" Order = "34" Size = "16" />
+          <Entry Name ="V19" Order = "35" Size = "16" />
+          <Entry Name ="V20" Order = "36" Size = "16" />
+          <Entry Name ="V21" Order = "37" Size = "16" />
+          <Entry Name ="V22" Order = "38" Size = "16" />
+          <Entry Name ="V23" Order = "39" Size = "16" />
+          <Entry Name ="V24" Order = "3a" Size = "16" />
+          <Entry Name ="V25" Order = "3b" Size = "16" />
+          <Entry Name ="V26" Order = "3c" Size = "16" />
+          <Entry Name ="V27" Order = "3d" Size = "16" />
+          <Entry Name ="V28" Order = "3e" Size = "16" />
+          <Entry Name ="V29" Order = "3f" Size = "16" />
+          <Entry Name ="V30" Order = "3f" Size = "16" />
+          <Entry Name ="V31" Order = "3f" Size = "16" />
+          <Entry Name ="fpsr" Order = "40" Size = "4" />
+          <Entry Name ="fpcr" Order = "41" Size = "4" />
+        </ExdiGdbServerRegisters>
+     </ExdiGdbServerConfigData>
+  </ExdiTarget>
+
+  <!-- BMC-OpenOCD HW debugger GDB server configuration -->
+  <ExdiTarget Name = "BMC-OpenOCD">
+    <ExdiGdbServerConfigData agentNamePacket = "BMC.OpenOCD.Windbg.Gdb" uuid = "72d4aeda-9723-4972-b89a-679ac79810ef" displayCommPackets = "yes" debuggerSessionByCore = "no" enableThrowExceptionOnMemoryErrors = "yes" qSupportedPacket="qSupported:xmlRegisters=aarch64,i386" >
+      <ExdiGdbServerTargetData targetArchitecture = "ARM64" targetFamily = "ProcessorFamilyARM64" numberOfCores = "1" EnableSseContext = "no" heuristicScanSize = "0xfffe" targetDescriptionFile = "target.xml" />
+      <GdbServerConnectionParameters MultiCoreGdbServerSessions = "no" MaximumGdbServerPacketLength = "1024" MaximumConnectAttempts = "3" SendPacketTimeout = "100" ReceivePacketTimeout = "3000">
+        <Value HostNameAndPort="10.124.204.127:3333" />

```

Comment:
Looks like this is an IP address from your testing. Can we put a placeholder in instead? Do we have anything tracking how this will work long term? E.g. can we pass parameters into the EXDI connection so that it can be specified as a parameter instead of in a config file?